dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      unset DO_COVERAGE
       make -C ../../ rubocop
```

Comment:
I would add all the linter tasks under the `unset DO_COVERAGE` in `*)` option, too (line 56).

By the way, it seems that `bundle exec rake haml_lint` was previously missed in `*)` option.